### PR TITLE
Merge document corrections in LBM

### PR DESCRIPTION
**Pull request by torao**
## Description

This PR is a document amendment to Ostracon's latest design changes.

- The role of "voter" and the election of voters have been removed.
- Minor wording changes to the English text and its application to the Japanese version.

See [fix/merge_docs_from_lbm](https://github.com/torao/ostracon/blob/fix/merge_docs_from_lbm/docs/en/01-overview.md) branch for a markdown view of this deliverable.

Closes: #XXX

### CODE REVIEW

File: docs/ja/01-overview.md [hunk 1]
```diff
@@ -1,89 +1,69 @@
...
-Validator に当選しているかをノード自身が判断します (すべてのノードが一斉にコイントスするのと似ています)。これは PoW のハッシュ計算で
-当選を引き当てる方法と比較して、大量の計算時間と電力消費を省略しつつ暗号論的な安全性を保証している優れた方法です。一方で、選出される
-Validator 数が決定的ではなく二項分布に従うランダムな振る舞い含むことや、当選ノード間の相互認識でプロトコルが複雑性が上がること、当選した
-にもかかわらず役割をサボタージュしたノードを見つけることができないこと、といったいくつかの理由により適用は難しいと判断しました。
+- Algorand も VRF を使用していますが我々とは全く異なる方法を採用しています。選挙開始時に各ノードが個別に VRF 乱数を生成し、その番号を用いて選挙が始まるとそれぞれのノードが VRF 乱数を生成して次の Validator に当選しているかを判断します (すべてのノードが一斉にコイントスするのと似ています)。これはハッシュ計算によって当選を特定する PoW 方式に比べて計算時間や消費電力を大幅に削減しつつ、暗号論的な安全性を保証することのできる優れた方法です。一方、この方式を LINE Blockchain に適用することはいくつかの理由で困難でした。まず選出される Validator 数が二項分布に従う非決定論的な結果になること。また当選したノード間の相互認識のプロトコルにより複雑さが増します。そして当選したにもかかわらず役割を果たさなかったノードを見つけることができません。
 
-他にいくつものコンセンサス機構について考慮しましたが、役割選出と合意アルゴリズムに関しては現在の選択が現実的に最良に近い選択と考えています。
-しかし、Ostracon は特定の研究理論に対しての実験的証明や実証実験をゴールとしていないため、将来的により良いアルゴリズムの提案があればそれを
-採用する準備があります。
+他のコンセンサスメカニズムも多数検討しましたが、P2P 分散システムにおける役割選出と合意アルゴリズムとしては現在の選択が最も現実的であると考えています。ただし、Ostracon は特定の研究理論に対する実験的な証明や実証を目的としていないため、将来、より優れたアルゴリズムが提案されればそれを採用する容易があります。
```
Comment (ulbqb) [inline docs/ja/01-overview.md:69]:
```suggestion
他のコンセンサスメカニズムも多数検討しましたが、P2P 分散システムにおける役割選出と合意アルゴリズムとしては現在の選択が最も現実的であると考えています。ただし、Ostracon は特定の研究理論に対する実験的な証明や実証を目的としていないため、将来、より優れたアルゴリズムが提案されれば採用する用意があります。
```

File: docs/ja/01-overview.md [hunk 2]
```diff
@@ -1,89 +1,69 @@
...
-*Bitcoin* や *Ethereum* で採用している PoW は最も有名なブロックチェーン向けコンセンサス機構です。これらはパブリックチェーンとして
-運用している実績がありますが、十分な時間が経過しないと結果が覆る可能性があるという機能的な制約を持ちます。これは、短期には lost update 問題を
-引き起こし、長期には必要なパフォーマンスが確保できないという問題が顕著に現れることから、PoW は検討初期の段階で選択肢から外れました。
+- Bitcoin や Ethereum (v1.10.22 以下) で採用されている **PoW** はブロックチェーンのコンセンサスメカニズムとしてもっとも有名なものです。これらはパブリックチェーンとして運用している実績がありますが、十分な時間が経過しないと整合性が保証されないという構造的な問題を持ちます。これは、短期には lost update 問題を引き起こし、長期には性能をスケールできないという問題が顕著に現れることから、PoW は我々の検討の初期段階で選択肢から外れました。
 
-*Tendermint* が合意アルゴリズムに採用している Tendermint-BFT はブロックチェーン向けによく考慮された設計です。短時間でファイナリティを
-保証できる点も我々の方針に適していました。一方で、選出アルゴリズムに採用している加重ラウンドロビンは決定論的に動作するため、誰でも将来の
-Proposer を知り得ることから標的を見つけて攻撃を準備しやすい点があります。このため Ostracon では攻撃の可能性を軽減する目的で VRF を使って
-予測不可能なアルゴリズムに置き換えています。
+- Tendermint のコンセンサスアルゴリズムである **Tendermint-BFT** はブロックチェーン向けによく考慮された設計です。短時間でファイナリティを保証できる点も我々の方針に適していました。一方、選出アルゴリズムの加重ラウンドロビンは決定論的に機能するため、誰でも将来の Proposer を予測して攻撃を準備することが容易になります。このため Ostracon では VRF を用いて選挙を予測不能にすることで Proposer への標的型攻撃の可能性を軽減しています。
```
Comment (ulbqb) [inline docs/ja/01-overview.md:65]:
```suggestion
- Tendermint のコンセンサスアルゴリズムである **Tendermint-BFT** はブロックチェーン向けによく考慮された設計です。短時間でファイナリティを保証できる点も我々の方針に適していました。一方、選出アルゴリズムの加重ラウンドロビンは決定論的に機能するため、誰でも将来の Proposer を予測して攻撃を準備することが容易になります。このため Ostracon では VRF を用いて選出を予測不能にすることで Proposer への標的型攻撃の可能性を軽減しています。
```

File: docs/ja/01-overview.md [hunk 3]
```diff
@@ -1,89 +1,69 @@
...
 
-## Consideration with Other Consensus Schemes
+## 他のコンセンサススキームの検討
 
 他のブロックチェーンではどのようなコンセンサス機構を採用しているのでしょうか? Ostracon の方向性を決定するために多くの比較と検討を行いました。
 
-*Bitcoin* や *Ethereum* で採用している PoW は最も有名なブロックチェーン向けコンセンサス機構です。これらはパブリックチェーンとして
-運用している実績がありますが、十分な時間が経過しないと結果が覆る可能性があるという機能的な制約を持ちます。これは、短期には lost update 問題を
-引き起こし、長期には必要なパフォーマンスが確保できないという問題が顕著に現れることから、PoW は検討初期の段階で選択肢から外れました。
+- Bitcoin や Ethereum (v1.10.22 以下) で採用されている **PoW** はブロックチェーンのコンセンサスメカニズムとしてもっとも有名なものです。これらはパブリックチェーンとして運用している実績がありますが、十分な時間が経過しないと整合性が保証されないという構造的な問題を持ちます。これは、短期には lost update 問題を引き起こし、長期には性能をスケールできないという問題が顕著に現れることから、PoW は我々の検討の初期段階で選択肢から外れました。
```
Comment (ulbqb) [inline docs/ja/01-overview.md:63]:
```suggestion
- Bitcoin や Ethereum (v1.10.22 以下) で採用されている **PoW** はブロックチェーンのコンセンサスメカニズムとしてもっとも有名なものです。これらはパブリックチェーンとして運用している実績がありますが、十分な時間が経過しないと整合性が保証されないという構造的な問題が存在します。これは、短期には lost update 問題を引き起こし、長期には性能をスケールできないという問題が顕著に現れることから、PoW は我々の検討の初期段階で選択肢から外されました。
```

File: docs/ja/01-overview.md [hunk 4]
```diff
@@ -1,89 +1,69 @@
...
-| Key Auth Protocol     | Station-to-Station             |                                                              |
-| Tx Sharing Protocol   | Gossiping                      | mempool                                                      |
-| Application Protocol  | ABCI                           |                                                              |
-| Interchain Protocol   | IBC (Cosmos Hub)               |                                                              |
-| Storage               | Embedded KVS                   | LevelDB                                                      |
-| Message Recovery      | WAL                            |                                                              |
-| Block Generation Time | 2 seconds                      |                                                              |
-
-## Ostracon Features
+まだブロックに取り込まれていないトランザクションは [mempool](03-tx-sharing.md) と呼ばれる Network レイヤーのアンチエントロピー機構 (ゴシッピング) によって各ノード間で共有されます。ここで、Network および Consensus レイヤーではトランザクションを単純なバイナリとして扱い、そのデータの内容には関与しません。
```
Comment (ulbqb) [inline docs/ja/01-overview.md:34]:
```suggestion
まだブロックに取り込まれていないトランザクションは [mempool](03-tx-sharing.md) と呼ばれる Network レイヤーのアンチエントロピー機構 (ゴシッピング) によって各ノード間で共有されます。ここで、Network および Consensus レイヤーではトランザクションを単純なバイナリとして扱い、そのデータの内容は解釈しません。
```

File: docs/ja/01-overview.md [hunk 5]
```diff
@@ -1,89 +1,69 @@
...
 
-ファイナリティとパフォーマンスの観点から Bitcoin のような Proof of Work よりも BFT (Byzantine Fault-Tolerance) に基づく P2P
-合意アルゴリズムの方が適しています。中でもブロックチェーンに最適化された近代的な設計が行われている Tendermint-BFT は我々の方針に最も
-近い実装でした (さらに良いことに Cosmos Hub とも接続できます)。
+BFT に基づく P2P (Peer to peer) コンセンサスアルゴリズムはファイナリティとパフォーマンスの面で Bitcoin のような Proof of Work (PoW) より我々の目的に適しています。中でもブロックチェーンに最適化された近代的な設計の Tendermint-BFT は我々の方針に最も近い実装でした。
 
-我々は我々のブロックチェーンをさらに改善するために Tendermint-BFT に新しい暗号技術を導入しています。この **検証可能な疑似乱数** (VRF) は
-ブロックを生成する Proposer ノードの選出にランダム性をもたせて未来の選出を予測困難にすることを目的に導入されました。このランダム性の導入により、
-悪意を持つ攻撃者に攻撃の猶予を与えたり、将来のある時点を狙って参加者どうしで共謀することを困難にする効果が期待できます。
+我々は、新しいブロックを生成するノードをランダムに選択する新しい暗号技術 Verifiable Random Function (VRF) を導入し、Tendermint-BFT ベースのブロックチェーンの改良を行っています。このランダム性は悪意のある攻撃を防ぐのに役立ち、将来のある時点を狙った参加者の共謀を困難にします。
```
Comment (ulbqb) [inline docs/ja/01-overview.md:21]:
```suggestion
我々は、新しいブロックを生成するノードをランダムに選択するために、新しい暗号技術 Verifiable Random Function (VRF) を導入し、Tendermint-BFT ベースのブロックチェーンの改良を行っています。このランダム性は悪意のある攻撃を防ぐのに役立ち、将来のある時点を狙った参加者の共謀を困難にします。
```

File: docs/ja/02-consensus.md [hunk 6]
```diff
@@ -2,55 +2,45 @@
...
-正しいかどうかを投票し、票は Tendermint-BFT によって他の Voter に複製され、すべての Voter 数の 2/3+1 以上の賛成票が集まればそのブロックは
-正式に*承認*されます。反対に、定足数の賛成票が集まらなければ提案されたブロックは拒否され新しいラウンドで選挙または投票からやり直しとなります
-(Tendermint-BFT には拒否の理由によってショートカットする経路がいくつかあります)。
+### ブロック生成
+
+選出された Proposer はブロックを提案します。まだブロックチェーンに追加されていない未承認のトランザクションはネットワーク上のノードに P2P で共有され、各ノードの mempool と呼ばれる領域に保管されます。Proposer に選ばれたノードは自分の mempool に残っている未承認のトランザクションからブロックを生成して Validator に提案します。
+
+### ブロック検証
+
+Validator は Proposer の提案したブロックを検証します。各 Voter はブロックの内容が正しいかどうかを投票し、Tendermint-BFT はその票を他の Validator に複製します。すべての Validator 数の 2/3 より多い賛成票が集まるとそのブロックは正式に承認されます。一方、定足数に達しない場合は提案されたブロックは却下され新たな選挙または投票がやり直しされます。
```
Comment (ulbqb) [inline docs/ja/02-consensus.md:21]:
```suggestion
Validator は Proposer の提案したブロックを検証します。各 Validator はブロックの内容が正しいかどうかを投票し、Tendermint-BFT はその票を他の Validator に複製します。すべての Validator 数の 2/3 より多い賛成票が集まるとそのブロックは正式に承認されます。一方、定足数に達しない場合は提案されたブロックは却下され新たに選挙と投票が行われます。
```

File: docs/ja/02-consensus.md [hunk 7]
```diff
@@ -60,60 +50,37 @@ t_h & = & {\rm vrf\_proof\_to\_hash}(\pi_h)
 \end{eqnarray*}
 ```
 
-VRF Proof $\pi$ を算出するためのメッセージ $m$ にはブロックそのもののハッシュ値は関与しないことに注意してください。ブロックのハッシュ値は
-ブロックを生成する Proposer が試行錯誤によって有利な値を導出できることから本質的に安全ではないと考えています。
+VRF Proof $\pi$ を算出するために使用されるメッセージ $m$ にはそのブロック自身のハッシュ値は含まれないことに注意してください。ブロックを生成する Proposer は試行錯誤によって有利な値を得ることができるため、ブロックのハッシュ値は本質的に反前ではないと考えています。
```
Comment (ulbqb) [inline docs/ja/02-consensus.md:53]:
```suggestion
VRF Proof $\pi$ を算出するために使用されるメッセージ $m$ にはそのブロック自身のハッシュ値は含まれないことに注意してください。ブロックを生成する Proposer は試行錯誤によって有利な値を得ることができるため、ブロックのハッシュ値から算出することは本質的に安全ではないと考えています。
```

**Review by ulbqb — COMMENTED** [review]
日本語版に関して、以下のものはどれかに統一したほうが良いと思いました。

- 合意メカニズム、合意アルゴリズム、コンセンサススキーム
- 未確定、未承認

File: docs/ja/01-overview.md [hunk 8]
```diff
@@ -1,89 +1,69 @@
...
-Proposer を知り得ることから標的を見つけて攻撃を準備しやすい点があります。このため Ostracon では攻撃の可能性を軽減する目的で VRF を使って
-予測不可能なアルゴリズムに置き換えています。
+- Tendermint のコンセンサスアルゴリズムである **Tendermint-BFT** はブロックチェーン向けによく考慮された設計です。短時間でファイナリティを保証できる点も我々の方針に適していました。一方、選出アルゴリズムの加重ラウンドロビンは決定論的に機能するため、誰でも将来の Proposer を予測して攻撃を準備することが容易になります。このため Ostracon では VRF を用いて選挙を予測不能にすることで Proposer への標的型攻撃の可能性を軽減しています。
 
-*Algorand* は我々とは大きく異なる方法で VRF を使用しています。Algorand では選挙が始まるとそれぞれのノードが VRF 乱数を生成して次の
-Validator に当選しているかをノード自身が判断します (すべてのノードが一斉にコイントスするのと似ています)。これは PoW のハッシュ計算で
-当選を引き当てる方法と比較して、大量の計算時間と電力消費を省略しつつ暗号論的な安全性を保証している優れた方法です。一方で、選出される
-Validator 数が決定的ではなく二項分布に従うランダムな振る舞い含むことや、当選ノード間の相互認識でプロトコルが複雑性が上がること、当選した
-にもかかわらず役割をサボタージュしたノードを見つけることができないこと、といったいくつかの理由により適用は難しいと判断しました。
+- Algorand も VRF を使用していますが我々とは全く異なる方法を採用しています。選挙開始時に各ノードが個別に VRF 乱数を生成し、その番号を用いて選挙が始まるとそれぞれのノードが VRF 乱数を生成して次の Validator に当選しているかを判断します (すべてのノードが一斉にコイントスするのと似ています)。これはハッシュ計算によって当選を特定する PoW 方式に比べて計算時間や消費電力を大幅に削減しつつ、暗号論的な安全性を保証することのできる優れた方法です。一方、この方式を LINE Blockchain に適用することはいくつかの理由で困難でした。まず選出される Validator 数が二項分布に従う非決定論的な結果になること。また当選したノード間の相互認識のプロトコルにより複雑さが増します。そして当選したにもかかわらず役割を果たさなかったノードを見つけることができません。
```
Comment (ulbqb) [inline docs/ja/01-overview.md:67]:
```suggestion
- Algorand も VRF を使用していますが我々とは全く異なる方法を採用しています。選挙開始時に各ノードが個別に VRF 乱数を生成し、その値を用いて、次の Validator に当選しているかを判断します (すべてのノードが一斉にコイントスするのと似ています)。これはハッシュ計算によって当選を特定する PoW 方式に比べて計算時間や消費電力を大幅に削減しつつ、暗号論的な安全性を保証することのできる優れた方法です。一方、この方式を LINE Blockchain に適用することはいくつかの理由で困難でした。まず選出される Validator 数が二項分布に従う非決定論的な結果になること。また当選したノード間の相互認識のプロトコルにより複雑さが増します。そして当選したにもかかわらず役割を果たさなかったノードを見つけることができません。
```

File: docs/en/02-consensus.md [hunk 9]
```diff
@@ -42,43 +50,36 @@ t_h & = & {\rm vrf\_proof\_to\_hash}(\pi_h)
...
+During the verification phase, in addition to the block verification, the following VRF-related verifications are performed:
 
-* The Proposer that generated the block must be a node selected based on the VRF hash of its previous block. This can be determined by matching the node that actually generated the block with the Proposer selected by weighted random sampling using the VRF hash $t$.
-* The $\pi$ contained in the block must be a VRF Proof generated using the private key of the Proposer. If the $t$ calculated from the VRF Proof $\pi$ matches the $t$ calculated using the `vrf_verify()` function, we can conclude that $\pi$ is not forged.
+* The proposer that generated the block must be a node selected based on the VRF hash of its previous block. This can be determined by matching the node that actually generated the block with the proposer selected by weighted random sampling using the VRF hash $t$.
+* The $\pi$ contained in the block must be a VRF Proof generated using the proposer's private key. If the $t$ computed from the VRF Proof $\pi$ matches the $t$ computed by the `vrf_verify()` function, we can conclude that $\pi$ is not forged.
 
-```math
+ ```math
```
Comment (ulbqb) [inline docs/en/02-consensus.md:66]:
```suggestion
```math
```

File: docs/en/03-tx-sharing.md [hunk 10]
```diff
@@ -2,30 +2,31 @@
 title: Transaction Sharing
 ---
 
-A client can send a transaction to any of the Ostracon nodes that joining the blockchain network. The transaction propagates to other Ostracon nodes and is ultimately shared by all Ostracon nodes.
+A client application can send a transaction to any Ostracon nodes that join the blockchain network. The transaction propagates to other Ostracon nodes and is ultimately shared by all Ostracon nodes.
 
 ## Mempool
 
-Once a block is accepted by the Ostracon consensus mechanism, the transactions contained in that block are considered *confirmed*. The unconfirmed transactions are validated stored in an area called [**mempool**](https://github.com/tendermint/tendermint/blob/v0.34.x/spec/abci/apps.md#mempool-connection), which is separate from the block storage, after validation such as signatures.
+<!-- Once a block is accepted by the Ostracon consensus mechanism, the transactions contained in that block are considered **confirmed**.  -->
```
Comment (ulbqb) [inline docs/en/03-tx-sharing.md:9]: Did you forget to remove it?

File: docs/en/02-consensus.md [hunk 11]
```diff
@@ -2,37 +2,45 @@
...
+
+## VRF-based proposer election
+
+VRF is an algorithm for generating a hash value $t$ that can be used as a cryptographic pseudorandom number. The difference between VRF and typical hash functions or pseudorandom number generators is that only the owner of the private key can generate the hash value $t$, and anyone with the corresponding public key can verify the correctness of the hash value.
+
+A VRF hash generator $k$ generates a proof $\pi$ (VRF Proof) from the message $m$ using its private key $S_k$ as in equation `1.`. Here the hash value $t$ can be obtained from the proof $pi$ using equation. `2.` On the other hand, to verify that the hash value $t$ was generated by the owner of the private key $S_k$ based on the message $m$, the verifier applies the public key $P_k$ for $S_k$, $m$, and $\pi$ to equation `3.` to verify that both hash values are identical.
 
 1. $\pi = {\rm vrf\\_prove}(S_k, m)$
 2. $t = {\rm vrf\\_proof\\_to\\_hash}(\pi)$
 3. ${\rm vrf\\_proof\\_to\\_hash}(\pi) \overset{\text{?}}{=} {\rm vrf\\_verify}(P_k, m, \pi)$
```
Comment (ulbqb) [inline docs/en/02-consensus.md:35]: How about writing something like:

```math
\begin{align*}
   \pi & = & {\rm vrf\\_prove}(S_k, m) \tag{1}\\
   t & = & {\rm vrf\\_proof\\_to\\_hash}(\pi) \tag{2}\\
   {\rm vrf\\_proof\\_to\\_hash}(\pi) & \overset{\text{?}} {=} & {\rm vrf\\_verify}(P_k, m, \pi) \tag{3}
\end{align*}
```

File: docs/ja/02-consensus.md [hunk 12]
```diff
@@ -2,55 +2,45 @@
...
 
-乱数の生成者 $k$ は式 `1.` のように自身の秘密鍵 $S_k$ を使ってメッセージ $m$ から証明 (VRF Proof) $\pi$ を生成します。ここでハッシュ値
-$t$ は式 `2.` を使って証明 $pi$ から生成することができます。一方、検証者はハッシュ値 $t$ が秘密鍵 $S_k$ の所有者によってメッセージ $m$ に
-基づいて生成されたものであることを検証するために、$S_k$ に対する公開鍵 $P_k$ と $m$, $\pi$ を式 `3.` に適用して同一のハッシュ値 $t$ が
-生成されることを確認します。
+VRF ハッシュの生成者 $k$ は式 `1.` のように自身の秘密鍵 $S_k$ を使ってメッセージ $m$ から証明 (VRF Proof) $\pi$ を生成します。ここでハッシュ値 $t$ は式 `2.` を使って証明 $pi$ から生成することができます。一方、検証者はハッシュ値 $t$ が秘密鍵 $S_k$ の所有者によってメッセージ $m$ に基づいて生成されたものであることを検証するために、式 `3.` に $S_k$ に対する公開鍵 $P_k$ と $m$, $\pi$ を適用して同一のハッシュ値 $t$ が生成されることを確認します。
 
 1. $\pi = {\rm vrf\\_prove}(S_k, m)$
 2. $t = {\rm vrf\\_proof\\_to\\_hash}(\pi)$
 3. ${\rm vrf\\_proof\\_to\\_hash}(\pi) \overset{\text{?}}{=} {\rm vrf\\_verify}(P_k, m, \pi)$
```
Comment (ulbqb) [inline docs/ja/02-consensus.md:35]: How about writing something like:

```math
\begin{align*}
   \pi & = & {\rm vrf\\_prove}(S_k, m) \tag{1}\\
   t & = & {\rm vrf\\_proof\\_to\\_hash}(\pi) \tag{2}\\
   {\rm vrf\\_proof\\_to\\_hash}(\pi) & \overset{\text{?}} {=} & {\rm vrf\\_verify}(P_k, m, \pi) \tag{3}
\end{align*}
```

File: docs/en/03-tx-sharing.md [hunk 13]
```diff
@@ -2,30 +2,31 @@
...
 
 ![Mempool in Ostracon structure](../static/tx-sharing/mempool.png)
 
-## Performance and Asynchronization
+## Performance and asynchronization
 
-Blockchain performance tends to focus on the speed of block generation, but in a practical system, the efficiency of sharing transactions among nodes is also an important factor that significantly affects overall performance. In particular, Ostracon's mempool must process a large number of transactions in a short period in exchange for Gossipping's network propagation speed.
+Blockchain performance tends to focus on the speed of block generation, but in a practical system, the efficiency of sharing transactions among nodes is also an important factor that significantly affects overall performance. For the high speed of Gossipping's network propagation, Ostracon's mempool must process a large number of transactions in a short period. <!-- in exchange for Gossipping's network propagation speed.-->
```
Comment (ulbqb) [inline docs/en/03-tx-sharing.md:21]:
```suggestion
Blockchain performance tends to focus on the speed of block generation, but in a practical system, the efficiency of sharing transactions among nodes is also an important factor that significantly affects overall performance. For the high speed of Gossipping's network propagation, Ostracon's mempool must process a large number of transactions in a short period.
```

**Review by kokeshiM0chi — APPROVED** [review]

File: docs/ja/01-overview.md (hunk 1 above)
Comment (torao) [reply docs/ja/01-overview.md]: fixed.

File: docs/ja/01-overview.md (hunk 2 above)
Comment (torao) [reply docs/ja/01-overview.md]: fixed.

File: docs/ja/01-overview.md (hunk 3 above)
Comment (torao) [reply docs/ja/01-overview.md]: fixed.

File: docs/ja/01-overview.md (hunk 4 above)
Comment (torao) [reply docs/ja/01-overview.md]: fixed.

File: docs/ja/01-overview.md (hunk 5 above)
Comment (torao) [reply docs/ja/01-overview.md]: fixed.

File: docs/ja/02-consensus.md (hunk 6 above)
Comment (torao) [reply docs/ja/02-consensus.md]: fixed.

File: docs/ja/02-consensus.md (hunk 7 above)
Comment (torao) [reply docs/ja/02-consensus.md]: fixed.

File: docs/ja/01-overview.md (hunk 8 above)
Comment (torao) [reply docs/ja/01-overview.md]: fixed. It was duplicated.

File: docs/en/02-consensus.md (hunk 9 above)
Comment (torao) [reply docs/en/02-consensus.md]: fixed.

File: docs/en/03-tx-sharing.md (hunk 10 above)
Comment (torao) [reply docs/en/03-tx-sharing.md]: The commented parts are those that were so in the LBM document. It's removed from this document.

File: docs/en/02-consensus.md (hunk 11 above)
Comment (torao) [reply docs/en/02-consensus.md]: I see that the auto-numbering doesn't work on GitHub (katex-based?), but `tag`s can be used. Thanks.

File: docs/ja/02-consensus.md (hunk 12 above)
Comment (torao) [reply docs/ja/02-consensus.md]: ditto. I replaced with a normal multiline formula style.

File: docs/en/03-tx-sharing.md (hunk 13 above)
Comment (torao) [reply docs/en/03-tx-sharing.md]: ditto.

**Comment by torao** [comment]
@ulbqb 
未承認 and コンセンサススキーム unified as 未確定 and 合意メカニズム respectively. But Algorithms are left separate as it's unnatural to treat them as essentially equivalent to Mechanisms.

**Review by tnasu — APPROVED** [review]

**Review by ulbqb — APPROVED** [review]

**Pull request merged**